pkg/actuators/machine/utils.go

```diff
@@ -277,3 +276,43 @@ func findAWSMachineProviderCondition(conditions []providerconfigv1.AWSMachinePro
 	}
 	return nil
 }
+
+// awsInstanceRegMatch represents Regex Match for AWS instance.
+var awsInstanceRegMatch = regexp.MustCompile("^i-[^/]*$")
+
+// mapToAWSInstanceID extracts the InstanceID from the KubernetesInstanceID
+// providerID represents the id for an instance in the kubernetes API;
+// the following form
+//  * aws:///<zone>/<awsInstanceId>
+func mapToAWSInstanceID(providerID string) (*string, error) {
```

Comment:
`s/mapToAWSInstanceID/getInstanceID` maybe? The word mapping evokes the idea there is a set of providerIDs that  are mapped to an instanceIDs. Which is correct from the mathematical point of view, though given you only parse a string, it's more suitable to use `get` or `extract` rather than `map`.